### PR TITLE
Return a more easily readable string from the Language's description property

### PR DESCRIPTION
**Pull request by ahoppen**

### CODE REVIEW

**Comment by ahoppen** [comment]
@swift-ci Please test

File: Sources/LanguageServerProtocol/SupportTypes/Language.swift [hunk 1]
```diff
@@ -37,10 +37,71 @@ public struct Language: RawRepresentable, Codable, Equatable, Hashable {
...
-extension Language: CustomStringConvertible {
-  public var description: String {
+extension Language: CustomStringConvertible, CustomDebugStringConvertible {
+  public var debugDescription: String {
     return rawValue
   }
+
+  public var description: String {
+    switch rawValue {
+    case "abap": return "ABAP"
```
Comment (benlangmuir) [inline Sources/LanguageServerProtocol/SupportTypes/Language.swift]: Why not switch on `self` (e.g. `case .abap`)?

Comment (ahoppen) [reply Sources/LanguageServerProtocol/SupportTypes/Language.swift]: Good question. Because I didn’t think about that. Changed it to switch on `self`.

**Comment by ahoppen** [comment]
@swift-ci Please test

**Pull request merged**